example/android/build.gradle

```diff
@@ -12,3 +12,14 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
     }
 }
+
+allprojects {
+    repositories {
+        maven {
+            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
+            url("${rootDir}/../node_modules/react-native/android")
```

Comment:
Nit: Do we have a more generic way to get the node_modules? For FURN I locally have `  url("$rootDir/../../../node_modules/react-native/android")` but I acknowledge our structure is a little messy.